server/database/repositories/TournamentRepository.ts

```diff
@@ -42,8 +46,10 @@ function toTournamentDb(t: TournamentWithoutMS) : TournamentDb {
     teams: t.teams.map((team) => toTeamDb(team)),
     matches: t.matches.map((match) => toMatchDb(match)),
     isFinished: t.isFinished,
+    type: t.type,
   };
 }
+
 function toTournament(tDoc: TournamentDocument) : TournamentWithoutMS {
   return {
     id: tDoc._id.toString(),
```

Comment:
Change this name to something more understandable `tDoc`

Reply:
This is code from the other task. I know these names aren't the best. I wrote about this. By the way of the correction task it should be changed @PrzemekRychter